utils/json_profiles_merger_lib.py

```diff
@@ -0,0 +1,173 @@
+"""A library that holds the bulk of the logic for merging JSON profiles.
+
+Collect median duration of events across these profiles.
+"""
+from __future__ import division
+
+import csv
+import gzip
+import json
+import os
+
+
+def _median(lst):
```

Comment:
https://docs.python.org/3/library/statistics.html#statistics.median ?

Reply:
Will replace with that once we're free of python 2